aip/general/0133.md

```diff
@@ -178,6 +178,10 @@ message CreateBookRequest {
     duplicate resource, the service **must** error with `PERMISSION_DENIED`
     instead.
 
+**Note:** When supplying a user-specified ID, while the ID is set on
+the request message for RPCs, for REST APIs, the `{resource}_id` field
+could be included as a query parameter when making the the request.
+
```

Comment:
```suggestion
**Note:** For REST APIs, the user-specified ID field, `{resource}_id`, 
will appear in the query parameters of the request.

```

I think we can tighten this up. No need to qualify "RPC" style APIs, we can just state "for this scenario, this will happen"